drivers/pds/lib/utils.go

```diff
@@ -481,11 +501,310 @@ func DeleteDeployment(deploymentID string) (*state.Response, error) {
+			dnsEndpoint = fmt.Sprint(value)
+			isfound = true
+		}
+	}
+	if !isfound {
+		logrus.Errorf("No connection string found")
+		return "", err
+	}
+
+	return dnsEndpoint, nil
+}
+
+//GetDeploymentCredentials returns the password to connect to the dataservice
+func GetDeploymentCredentials(deploymentID string) (string, error) {
+	dataServiceDeployment := components.DataServiceDeployment
+	dataServicePassword, err := dataServiceDeployment.GetDeploymentCredentials(deploymentID)
+	if err != nil {
+		logrus.Errorf("An Error Occured %v", err)
+		return "", err
+	}
+	pdsPassword := dataServicePassword.GetPassword()
+	return pdsPassword, nil
+}
+
+//CreatecassandraWorkload generate workloads on the cassandra db
+func CreatecassandraWorkload(cassCommand string, deploymentName string, namespace string) (*v1.Deployment, error) {
+
+	var replicas int32 = 1
+	deploymentSpec := &v1.Deployment{
+		ObjectMeta: metav1.ObjectMeta{
+			GenerateName: deploymentName + "-",
+			Namespace:    namespace,
+			Labels:       map[string]string{"app": deploymentName},
+		},
+		Spec: v1.DeploymentSpec{
+			Replicas: &replicas,
+			Selector: &metav1.LabelSelector{
+				MatchLabels: map[string]string{"app": deploymentName},
+			},
+			Template: corev1.PodTemplateSpec{
+				ObjectMeta: metav1.ObjectMeta{
+					Labels: map[string]string{"app": deploymentName},
+				},
+				Spec: corev1.PodSpec{
+					Containers: []corev1.Container{
+						{
+							Name:    deploymentName,
+							Image:   "scylladb/scylla:4.1.11",
+							Command: []string{"/bin/sh", "-c", cassCommand},
+						},
+					},
+					RestartPolicy: "Always",
+				},
+			},
+		},
+	}
+	deployment, err := k8sApps.CreateDeployment(deploymentSpec, metav1.CreateOptions{})
+	if err != nil {
+		logrus.Errorf("An Error Occured while creating deployment %v", err)
+		return nil, err
+	}
+
+	err = k8sApps.ValidateDeployment(deployment, timeOut, timeInterval)
+	if err != nil {
+		logrus.Errorf("An Error Occured while validating the pod %v", err)
+		return nil, err
+	}
+
+	//TODO: Remove static sleep and verify the injected data
+	time.Sleep(1 * time.Minute)
+
+	return deployment, nil
+}
+
+//CreatepostgresqlWorkload generate workloads on the pg db
+func CreatepostgresqlWorkload(dnsEndpoint string, pdsPassword string, scalefactor string, iterations string, deploymentName string, namespace string) (*v1.Deployment, error) {
+	var replicas int32 = 1
+	deploymentSpec := &v1.Deployment{
+		ObjectMeta: metav1.ObjectMeta{
+			GenerateName: deploymentName + "-",
+			Namespace:    namespace,
+		},
+		Spec: v1.DeploymentSpec{
+			Replicas: &replicas,
+			Selector: &metav1.LabelSelector{
+				MatchLabels: map[string]string{"app": deploymentName},
+			},
+			Template: corev1.PodTemplateSpec{
+				ObjectMeta: metav1.ObjectMeta{
+					Labels: map[string]string{"app": deploymentName},
+				},
+				Spec: corev1.PodSpec{
+					Containers: []corev1.Container{
+						{
+							Name:    "pgbench",
+							Image:   "madan19/pgbench:pgloadTest1",
```

Comment:
same here. if this is in ur repo move it to portworx

Reply:
can you please share me the repo name where can I push this image to ?

Reply:
done, as we discussed I have moved the image under portworx org